jsonrpc.go

```diff
@@ -1047,6 +1061,8 @@ var rpcHandlers = map[string]RPCHandler{
 	"renewDHCPLease":         {Func: rpcRenewDHCPLease},
 	"keyboardReport":         {Func: rpcKeyboardReport, Params: []string{"modifier", "keys"}},
```

Comment:
Now returns the KeysDownState as well, browser clients that aren't expecting that response will silently ignore it.